Firestore/tests/Unit/QueryTest.php

```diff
@@ -35,6 +35,8 @@
 use PHPUnit\Framework\TestCase;
 use Prophecy\Argument;
 use Prophecy\PhpUnit\ProphecyTrait;
+use Yoast\PHPUnitPolyfills\Polyfills\ExpectException;
```

Comment:
Merge with `main` and remove this, its removed now to drop support for php<=7.3

Reply:
Done!

Reply:
Still present

Reply:
Oops, thank you for pointing this out. Changed now,.